### PR TITLE
chore(linux): Rename (lib)kmnkbp to (lib)keymancore 〽️

### PR DESCRIPTION
**Pull request by ermshiperete**
Part of #9733.

@keymanapp-test-bot skip

### CODE REVIEW

File: core/src/version.rc [hunk 1]
```diff
@@ -23,7 +23,7 @@
         VALUE "InternalName", "Keyman Core"
         VALUE "LegalCopyright", "© SIL International"
         VALUE "LegalTrademarks", ""
-        VALUE "OriginalFilename", "KMNCORE0-0.DLL"
+        VALUE "OriginalFilename", "KEYMANCORE1-0.DLL"
```
Comment (mcdurdin) [inline core/src/version.rc:26]: Here we have KEYMANCORE1-0.DLL, but in kmdev.wxs and elsewhere we have keymancore1.dll.

```suggestion
        VALUE "OriginalFilename", "KEYMANCORE1.DLL"
```

File: linux/debian/libkmnkbp0-0.symbols [hunk 2]
```diff
@@ -1,43 +1,43 @@
-libkmnkbp0.so.0 libkmnkbp0-0 #MINVER#
+libkeymancore.so.1 libkmnkbp0-0 #MINVER#
```
Comment (mcdurdin) [inline linux/debian/libkmnkbp0-0.symbols:1]: Is this still supposed to be libkmnkbp0-0 here? And noting the filename is still given as libkmnkbp0-0.symbols.

**Review by mcdurdin — APPROVED** [review]
LGTM but now I am wondering why we have both keyman-engine.vcxproj which appears to link keymancore.lib and keyman32.vcxproj which has links libkeymancore.a? (@rc-swag?)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.197-alpha](https://keyman.com/downloads/releases/alpha/17.0.197)

**Pull request merged**